### ISSUE TITLE
[BUG][NOWEB] - Issues encountered in NOWEB 12.6

### ISSUE THREAD

**Issue by nachmanmetodi** [allburov/whatsapp-http-api#708]
Failed functionality:
- [x] Buttons don't work but trigger WhatsApp Web notifications (no phone notifications)
- [x] Webhooks only trigger with engine.event, not with individual events (except session/message updates)
- [x] No success/failure response when adding participants during group creation
- [x] Media loading times out consistently

API Endpoint issues:
- [x] /contacts/about returns 422 "not implemented"
- [x] /contacts/profile-picture returns encrypted file link only
- [x] /contacts/block and /contacts/unblock return 422 "not implemented"

[![patron:PLUS](https://img.shields.io/badge/patron-PLUS-a0e6ba)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
Hi! ~~What engine do you use?~~ I see, NOWEB

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
Is there any docker logs from the start of container and when you start a session may be?

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
> Buttons don't work but trigger WhatsApp Web notifications (no phone notifications)

Buttons are fragile creatures, no guarantee it'll always work :( Looks like Meta got patched our solution recently...
Consider having a fallback logic to regular text in case of such errors.
We'll see what we can do with buttons 🤞

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
> Webhooks only trigger with engine.event, not with individual events (except session/message updates)

Which event exactly do you want to listen? 

`engine.event` is internal engine events, it can be too wordy sometimes, but most of them mapped to WAHA events https://waha.devlike.pro/docs/how-to/webhooks/#events
Just let us know which event doesn't works and we'll double check! 
Tested it with `labels.*` - works fine

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
> No success/failure response when adding participants during group creation

https://github.com/devlikeapro/waha/issues/709

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
>  Media loading times out consistently

Could you explain it, when does it happen? What you sending a message or receiving?

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
>  /contacts/about returns 422 "not implemented"

Yeah, that's true, NOWEB doesn't support it now :( Consider creating a feature request!


[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
>  /contacts/profile-picture returns encrypted file link only

that's weird, just tested it - works fine. It could return ".enc" file, but if you open as regular jpeg - it does show the picture

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
> /contacts/block and /contacts/unblock return 422 "not implemented"

Mistake in the doc, it's not supported, yeah :( Feel free to open [feature request](https://github.com/devlikeapro/waha/issues/new/choose)!

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
If you could open issue for each real bug / feature request - that'd be awesome! :pray: Hard to keep track of all changes in one big issue :) 

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
